tools/template_based_query_generation/src/main/java/Keywords.java

```diff
@@ -0,0 +1,23 @@
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * Keywords set interface that all keyword set classes implement
+ */
+
+public interface Keywords {
```

Comment:
I believe we were expecting that these classes would allow Allen and I to work separately on DDL and DML respectively, but we've divided the work in a different way so yes I think it makes sense to just use the Keywords class and specify file_path. Will make that change soon.